socorro/unittest/external/es/test_supersearch.py

```diff
@@ -1752,16 +1713,13 @@ def test_get_with_number_histogram(self):
         version_facet = (
             res['facets']['histogram_build_id'][0]['facets']['version']
         )
-        eq_(len(version_facet), number_of_crashes)
+        assert len(version_facet) == number_of_crashes
 
         # Test errors
         args = {}
         args['_histogram.build_id'] = ['unknownfield']
-        assert_raises(
-            BadArgumentError,
-            self.api.get,
-            **args
-        )
+        with pytest.raises(BadArgumentError):
```

Comment:
I really like the concision of this

Reply:
Me, too. Plus the function call isn't perverted which makes it easier (for me) to understand what's going on.